### PR TITLE
Update Callout component style

### PR DESCRIPTION
**Pull request by roadlittledawn**
Closes [newrelic/docs-website#649](https://github.com/newrelic/docs-website/issues/649)

## Description

Updates Callout component style to more align with current docs site style. I added small border radius because sharp corners looked a little weird.

### CODE REVIEW

**Comment by jerelmiller** [comment]
Oooo these look really nice!

File: packages/gatsby-theme-newrelic/src/components/Callout.js
Comment (zstix) [inline packages/gatsby-theme-newrelic/src/components/Callout.js:65]: We can probably delete this line.
```suggestion
```

File: packages/gatsby-theme-newrelic/src/components/Callout.js
Comment (jerelmiller) [inline packages/gatsby-theme-newrelic/src/components/Callout.js:21]: I think there is an opportunity for consolidation here. Rather than listing all 4 of these as separate properties, I'd use an override for the thicker border. This would conslidate the styles to:

```
border: 1px solid var(--color-red-400);
border-left-width: 6px;
```

Use this technique for the other variants as well.

File: packages/gatsby-theme-newrelic/src/components/Callout.js
Comment (jerelmiller) [inline packages/gatsby-theme-newrelic/src/components/Callout.js:55]: Could we use `0.25rem`? That border radius is pretty standard across our components with a border radius.

File: packages/gatsby-theme-newrelic/src/components/Callout.js
Comment (roadlittledawn) [reply packages/gatsby-theme-newrelic/src/components/Callout.js:65]: oops. made the change but didn't commit it. thanks!

File: packages/gatsby-theme-newrelic/src/components/Callout.js
Comment (jerelmiller) [inline packages/gatsby-theme-newrelic/src/components/Callout.js:35]: Now that I see this, there is one more bit of conslidation we could probably do 😄 . Since we know all variants will have the same border, just with different border colors, we could extract the common bit to the main style below and have this only change the border color.

```
[VARIANTS.CAUTION]: css`
  // ...
  border-color: var(--color-red-400);
`,
[VARIANTS.TIP]: css`
  // ...
  border-color: var(--color-green-400);
`
```

Further down:

```
css={css`
  // ...
  border: 1px solid;
  border-left-width: 6px;
  ${styles.variant[variant]}
`}
```

**Review by jerelmiller — COMMENTED** [review]
Added one more suggestion. I'll leave it up to you if you want to address it or not.

Looks like you've got a merge conflict. I'd be happy to approve once you're able to fix that!

File: packages/gatsby-theme-newrelic/src/components/Callout.js
Comment (roadlittledawn) [reply packages/gatsby-theme-newrelic/src/components/Callout.js:35]: oh yes indeed. good call.

**Comment by roadlittledawn** [comment]
@jerelmiller just resolved the merge conflict and pushed back up.

File: packages/gatsby-theme-newrelic/src/components/Callout.js
Comment (jerelmiller) [inline packages/gatsby-theme-newrelic/src/components/Callout.js:46]: This margin should be gone. It was [removed](https://github.com/newrelic/gatsby-theme-newrelic/pull/232/files#diff-dfea6acf43189bba059439ab9d704e1ca7ff214d6ed7a1d52ba176c9dbb8920dL36) in favor of adding it to the [MDX variant of this component](https://github.com/newrelic/gatsby-theme-newrelic/pull/232/files#diff-4a9b5570faaa9d3542fba8b4d8ba714c5f163bf3fa25c12a34af75e41a8a28f2)

Comment (roadlittledawn) [reply packages/gatsby-theme-newrelic/src/components/Callout.js:46]: d'oh! doing that now.

Comment (jerelmiller) [reply packages/gatsby-theme-newrelic/src/components/Callout.js:46]: Thats the last thing I promise!! Thanks for doing this though. They look 🔥 🔥 🔥

**Review by jerelmiller — APPROVED** [review]

File: packages/gatsby-theme-newrelic/src/components/Callout.js
Comment (roadlittledawn) [reply packages/gatsby-theme-newrelic/src/components/Callout.js]: haha. NP! appreciate the 🦅 👁️

**Pull request merged**
